### PR TITLE
[139] Product Page using HTML

### PR DESCRIPTION
**Pull request by damlois**
This PR contains Implementation for my product page task using HTML.

See screenshots below.
image
image

### CODE REVIEW

File: submissions/loisAdegbohungbe/HTML/product.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    <footer>
+        <p>Contact us on</p>
+        <ul>
+            <li><a href="https://www.twitter.com">Twitter</a></li>
+            <li><a href="https://www.linkedin.com">LinkedIn</a></li>
+            <li><a href="https://www.medium.com">Medium</a></li>
+        </ul>
+        <p>&copy; 2020 Louissa Store</p>
+    </footer>
```
Comment (Ireti203) [inline submissions/loisAdegbohungbe/HTML/product.html:47]: Format the footer tag properly.

File: submissions/loisAdegbohungbe/HTML/product.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Maxi 4-Burner Gas Cooker | Louissa Store</title>
+</head>
+<body>
+    <h2>Maxi 4-Burner Gas Cooker</h2>
+
```
Comment (Ireti203) [inline submissions/loisAdegbohungbe/HTML/product.html:9]: Remove whitespace.

File: submissions/loisAdegbohungbe/HTML/product.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    <!--This section is for product images and price-->
+    <section>
+        <div>
+            <img src="images/image1.jpg" width="300" alt="image1"/>
+            <img src="images/image2.jpg" width="300" alt="image2"/>
+        </div>
+        <h1>Price: &#8358;50,400</h1>
+    </section>
+
```
Comment (Ireti203) [inline submissions/loisAdegbohungbe/HTML/product.html:18]: Remove whitespace

File: submissions/loisAdegbohungbe/HTML/product.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    <section>
+        <h3>Product Description</h3>
+        <p>Maxi 4-Burner is a omfortable and easy to use gas cooker. 
+            Cook your meals faster with this appliance and enjoy neatness and ease.
+            Its durability is sure and getting this, you have made a rich step to cooking your delicious meals.
+        </p>
+    </section>
+
+    <!--This navigation section displays other related products-->
+    <nav>
```
Comment (Ireti203) [inline submissions/loisAdegbohungbe/HTML/product.html:29]: Why are you using a nav tag here? Nav tags are used for major blocks of navigation links. You can use a section tag instead

File: submissions/loisAdegbohungbe/HTML/product.html [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+    <!--This section is for product description-->
+    <section>
+        <h3>Product Description</h3>
+        <p>Maxi 4-Burner is a omfortable and easy to use gas cooker. 
+            Cook your meals faster with this appliance and enjoy neatness and ease.
+            Its durability is sure and getting this, you have made a rich step to cooking your delicious meals.
+        </p>
+    </section>
+
```
Comment (Ireti203) [inline submissions/loisAdegbohungbe/HTML/product.html:27]: Remove whitespace.

File: submissions/loisAdegbohungbe/HTML/product.html [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    <nav>
+        <p>You may also like</p>
+        <ol>
+            <li><a href="#">Scanfrost Gas Cooker</a></li>
+            <li><a href="#">Nexus 4-Burner Gas Cooker</a></li>
+            <li><a href="#">Maxi 60*60 Standing Cooker 3 Gas 1 Electric</a></li>
+            <li><a href="#">THERMOCOOL GAS COOKER MY DIVA 604G OG-6840 INX</a></li>
+        </ol>
+    </nav>
+
```
Comment (Ireti203) [inline submissions/loisAdegbohungbe/HTML/product.html:38]: Remove whitespace

**Review by Ireti203 — CHANGES_REQUESTED** [review]

File: submissions/loisAdegbohungbe/HTML/product.html (hunk 1 above)
Comment (damlois) [reply submissions/loisAdegbohungbe/HTML/product.html:47]: How do I do that @Ireti203 .

**Review by Ireti203 — APPROVED** [review]

**Pull request merged**